http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/BaseHttpAdvancedIT.java

```diff
@@ -186,13 +185,9 @@ public void nonAppRedirections() {
     }
 
     @Test
-    @Disabled("blocked by: https://issues.redhat.com/browse/QUARKUS-781")
-    public void microprofileHttpClientRedirection() throws Exception {
-        HealthClientService healthHttpClient = new RestClientServiceBuilder<HealthClientService>(getAppEndpoint())
-                .withHostVerified(true).withPassword(PASSWORD).withKeyStorePath(KEY_STORE_PATH)
-                .build(HealthClientService.class);
-
-        assertThat(HttpStatus.SC_OK, equalTo(healthHttpClient.health().getStatus()));
+    public void microprofileHttpClientRedirection() {
+        io.restassured.response.Response health = getApp().given().get("api/client");
```

Comment:
please avoid full qualified names if are not required.

Reply:
It is required in this case, this class also uses another Response class[1].

[1] https://github.com/quarkus-qe/quarkus-test-suite/blob/9963984c74f67f8d05c2603abf8204a894c7558e/http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java#L49